vendor/deps/cli-ui/lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -334,22 +334,15 @@ def stop_line_select
         end
 
         def read_char
-          raw_tty! do
```

Comment:
Did we do a full import of the latest `cli-ui`? Or is this a hand-edit?

Reply:
🤦‍♀️